dkg/sync/client.go

```diff
@@ -51,7 +51,7 @@ func (*Client) Shutdown() error {
 }
 
 // NewClient starts a goroutine that establishes a long lived connection to a p2p server and returns a new Client instance.
-func NewClient(ctx context.Context, tcpNode host.Host, server p2p.Peer, hash []byte, onFailure func(), ch chan *pb.MsgSyncResponse) Client {
+func NewClient(ctx context.Context, tcpNode host.Host, server p2p.Peer, hashSig []byte, onFailure func(), ch chan *pb.MsgSyncResponse) Client {
```

Comment:
discussion: what does `hashSig` mean? signature on hash, `sigHash`?

Reply:
hashSig is a signature of config_hash signed using charon-enr-private-key